### PR TITLE
Opt out of rollback on reorg for templates

### PR DESCRIPTION
**Pull request by JonoPrest**
Currently it incurs a performance hit on historical sync. For starting up and development we want to encourage people to turn it off until they deploy for production.

### CODE REVIEW

File: codegenerator/cli/npm/envio/evm.schema.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
       ]
     },
     "rollback_on_reorg": {
-      "description": "A flag to indicate if the indexer should rollback to the last known valid block on a reorg (default: true)",
+      "description": "A flag to indicate if the indexer should rollback to the last known valid block on a reorg. This currently incurs a performance hit one historical sync and is recommended to to turn this off while developing (default: true)",
```
Comment (moose-code) [inline codegenerator/cli/npm/envio/evm.schema.json:72]:
```suggestion
      "description": "A flag to indicate if the indexer should rollback to the last known valid block on a reorg. This currently incurs a performance hit on historical sync and is recommended to turn this off while developing (default: true)",
```

File: codegenerator/cli/src/config_parsing/human_config.rs [hunk 2]
```diff
@@ -146,7 +146,9 @@ pub mod evm {
         #[serde(skip_serializing_if = "Option::is_none")]
         #[schemars(
             description = "A flag to indicate if the indexer should rollback to the last known \
-                           valid block on a reorg (default: true)"
+                           valid block on a reorg. This currently incurs a performance hit one \
+                           historical sync and is recommended to to turn this off while \
+                           developing (default: true)"
```
Comment (moose-code) [inline codegenerator/cli/src/config_parsing/human_config.rs:151]:
```suggestion
                           valid block on a reorg. This currently incurs a performance hit on \
                           historical sync and is recommended to turn this off while \
                           developing (default: true)"
```

File: codegenerator/cli/templates/static/erc20_template/javascript/config.yaml [hunk 3]
```diff
@@ -11,3 +11,6 @@ networks:
         events:
           - event: "Approval(address indexed owner, address indexed spender, uint256 value)"
           - event: "Transfer(address indexed from, address indexed to, uint256 value)"
+# Rollback on reorg mode currently incurs a performance hit on historical sync
+# while developing an indexer we recommend setting this to false
+rollback_on_reorg: false
```
Comment (moose-code) [inline codegenerator/cli/templates/static/erc20_template/javascript/config.yaml:16]: Nicely put 💪

**Review by moose-code — APPROVED** [review]
Just a couple spelling change suggestions! 

lots of liniting changes it seems, glossed over. 

Nice!!

File: codegenerator/cli/src/config_parsing/human_config.rs (hunk 2 above)
Comment (JonoPrest) [reply codegenerator/cli/src/config_parsing/human_config.rs]: Thanks JJ, I copied this in locally and ran the generator to output the schema 👍🏼

File: codegenerator/cli/templates/static/erc20_template/javascript/config.yaml (hunk 3 above)
Comment (JonoPrest) [reply codegenerator/cli/templates/static/erc20_template/javascript/config.yaml:16]: Cool just to note, this message won't be in contract import templates.

**Pull request merged**